#!/usr/bin/env python
PACKAGE = "proc_mapping_config_server"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("minIntensity", double_t, 0, "Minimum intensity.", 0.07, 0, 1)
gen.add("maxIntensity", double_t, 0, "Maximum intensity.", 1.0, 0, 1)
gen.add("minRange", double_t, 0, "Minimum range.", 0.1, 0, 100)
gen.add("maxRange", double_t, 0, "Maximum range.", 100.0, 0, 100)

exit(gen.generate(PACKAGE, "proc_mapping_config_server", "ProcMappingServer"))